### ISSUE TITLE
Screenshots are missing in the donut report

### ISSUE THREAD

**Issue by nithingg** [MagenTys/donut-maven-plugin#19]
After upgrading to the latest version (1.2.2), screenshots are missing in the report html. Prior to the update, screenshots were getting attached to the failed scenarios.

Donut Configuration

![image](https://user-images.githubusercontent.com/43971386/58523709-9a896680-8219-11e9-912f-0853b6ee042b.png)

Cucumber Dependency

![image](https://user-images.githubusercontent.com/43971386/58523690-8e9da480-8219-11e9-8d11-6b9e35c90f02.png)

The failed screenshots are available in the cucumber report folder. However, donut is failing to attach it to the report. 

Console Log

[INFO] --- donut-maven-plugin:1.2.2:generate (reporting) @ XXX_Automation ---
[INFO] Generating reports...
[WARNING] Could not lookup path List(projectMetadata, projectVersion) in List(Identifier(List(projectMetadata, projectVersion)))
[INFO] Donuts created at: C:\XXX_Automation\target\donut/donut-report.html

**Comment by jnernest** [comment]
Same issues persist for me too. JSON contains image embedded whereas in Donut the image is missing

**Comment by jnernest** [comment]
screenshots are embedded under after tag in JSON Object earlier it used to be under result, I think we need to code as if to read after and before tag in JSON Objects

**Comment by jnernest** [comment]
@After and @Before are now available on JSON need to parsed.

**Comment by jnernest** [comment]
@chdask @davebassan can anyone work on this as we are not able to see screenshots on the reports, can you please see to parse the JSON with after and show on report

**Comment by jnernest** [comment]
can anyone pick this up...becoz screenshots are missing causes major issue.
@davebassan @temyers @amitsha @mamalisk @Hamo69

**Comment by ravikalaa-krishnan** [comment]
Thanks jnernest for the fix. We've found workaround is to lower the cucumber-junit version to info.cukes (from io.cucumber) , tends to attach the screenshots in the donut reports.

**Comment by jnernest** [comment]
@ravikalaa-krishnan  Yes we were using info.cukes earlier and it was working fine now after upgrade to io.cucumber we had issues. I have made changes to code to show screenshot after upgrading to io.cucumber.

i have the jar at https://github.com/jnernest/donut-screenshot-for-io.cucumber

Try and let me know is there any issues.

**Comment by ravikalaa-krishnan** [comment]
@jnernest Thanks for the jar. I ll give it a try. :-)

**Comment by sspriya4789** [comment]
@jnernest I am getting **error in opening zip file** error when i add it via dependency and tried
